src/pages/FeatureRequests.jsx

```
@@ -0,0 +1,191 @@
+            <NavBar />
+            <div className="container mx-auto max-w-[1280px] px-6">
+                <h1 className="text-left text-4xl font-bold pt-12 pb-6">Suggest a feature</h1>
+                <p className="font-medium">
+                    Tell us what you’d like to see on {import.meta.env.VITE_APP_NAME}
+                </p>
+                <div className="flex flex-wrap md:flex-nowrap gap-6 pt-12 pb-32">
+                    <div className="bg-content1 p-4 rounded-xl border-1 border-bg-content2 basis-1/3 grow h-full">
+                        <div className="space-y-2">
+                            <h2 className={`text-lg font-semibold`}>Add your idea</h2>
+                            <p className="text-default-600">
+                                This could be a problem you're having or something you wish was
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)